ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -625,6 +638,22 @@ const DataSetPage = fp.flow(withUserProfile(), withCurrentWorkspace(), withUrlPa
       return ids.map(id => setsById.get(id));
     }
 
+    getPrePackagedList() {
+      let prepackagedList = Object.keys(PrepackagedConceptSet);
+      if (!getCdrVersion(this.props.workspace, this.props.cdrVersionListResponse).hasFitbitData) {
+        // TO DO Fix spelling mistake in prepackageSet
```

Comment:
Is there a ticket for this? If so, could you include it in this comment?

Reply:
Fixed in this PR